{
    "CVE_data_meta": {
        "ASSIGNER": "cve@mitre.org",
        "ID": "CVE-2019-10904",
        "STATE": "PUBLIC",
        "UPDATED": "2020-06-15"
    },
    "cna-container": {
        "affected": {
            "vendors": [
                {
                    "products": [
                        {
                            "versions": [
                                {
                                    "version_value": "n/a"
                                }
                            ]
                        }
                    ],
                    "vendor_name": "n/a"
                }
            ]
        },
        "descriptions": [
            {
                "lang": "eng",
                "value": "Roundup 1.6 allows XSS via the URI because frontends/roundup.cgi and roundup/cgi/wsgi_handler.py mishandle 404 errors."
            }
        ],
        "problemtypes": {
            "descriptions": [
                [
                    {
                        "description": "n/a",
                        "lang": "eng",
                        "type": "text"
                    }
                ]
            ]
        },
        "references": [
            {
                "name": "https://github.com/python/bugs.python.org/issues/34",
                "refsource": "MISC",
                "url": "https://github.com/python/bugs.python.org/issues/34"
            },
            {
                "name": "https://bugs.python.org/issue36391",
                "refsource": "MISC",
                "url": "https://bugs.python.org/issue36391"
            },
            {
                "name": "https://www.openwall.com/lists/oss-security/2019/04/05/1",
                "refsource": "MISC",
                "url": "https://www.openwall.com/lists/oss-security/2019/04/05/1"
            },
            {
                "name": "[oss-security] 20190407 Re: XSS in roundup bug tracker 404 page",
                "refsource": "MLIST",
                "url": "http://www.openwall.com/lists/oss-security/2019/04/07/1"
            },
            {
                "name": "[debian-lts-announce] 20190407 [SECURITY] [DLA 1750-1] roundup security update",
                "refsource": "MLIST",
                "url": "https://lists.debian.org/debian-lts-announce/2019/04/msg00009.html"
            }
        ]
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "5.0"
}